cmd/dep/vndr_importer_test.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"log"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/golang/dep"
+	"github.com/golang/dep/internal/gps"
+	"github.com/golang/dep/internal/test"
+	"github.com/pkg/errors"
+)
+
+func TestVndrConfig_Import(t *testing.T) {
```

Comment:
My apologies for not giving a heads up that this was incoming before you started... Another PR was just merged which switched the importers to table based tests. Below is an example of how the tests should look now. I hope it's not too much trouble to update the tests here to match, let me know if they don't line up well, I see you are testing slightly differently (using deep equal). 

https://github.com/golang/dep/blob/044c4ef93b6a8648f5d07f6f099b9e5287a8dd23/cmd/dep/godep_importer_test.go#L21-L49